estate/__manifest__.py

```diff
@@ -15,5 +15,10 @@
         "views/estate_property_views.xml",
         "views/estate_menus.xml",
         "views/res_users_views.xml",
+        "data/estate.property.type.csv",
```

Comment:
1. Normally we use XML files to create module data. But I suppose the documentation adds it with a CSV file for practicing purposes. 
2. This file should be named with `_`

```suggestion
        "data/estate_property_type_data.csv",
```

Please, check the guidelines for [file naming ](https://github.com/OCA/odoo-community.org/blob/master/website/Contribution/CONTRIBUTING.rst#15file-naming) and apply it to these files.